rootfs-configs.yaml

```diff
@@ -143,4 +143,6 @@ rootfs_configs:
     arch_list:
       - amd64
       - arm64
+      - armel
```

Comment:
Actually, `armel` is not really needed.  We can run LTP with `armhf` on all the 32-bit arm platforms we have.  That's what the other Buster rootfs images do.

Reply:
okay  updated the patch.